Bow.xcodeproj/project.pbxproj

```diff
@@ -218,8 +218,11 @@
 		11F41CA2231818DB00BD1E87 /* BracketLaws.swift in Sources */ = {isa = PBXBuildFile; fileRef = 11F41C9E231818AE00BD1E87 /* BracketLaws.swift */; };
 		11F41CA3231818DB00BD1E87 /* MonadDeferLaws.swift in Sources */ = {isa = PBXBuildFile; fileRef = 11F41C9F231818AF00BD1E87 /* MonadDeferLaws.swift */; };
 		600D30432352C08F00F7B64B /* Semigroupal.swift in Sources */ = {isa = PBXBuildFile; fileRef = 600D30422352C08F00F7B64B /* Semigroupal.swift */; };
-		600D30452352D03400F7B64B /* SemigroupalLaws.swift in Sources */ = {isa = PBXBuildFile; fileRef = 600D30442352D03400F7B64B /* SemigroupalLaws.swift */; };
-		601914CC2352F2B5007ACE81 /* TupleN.swift in Sources */ = {isa = PBXBuildFile; fileRef = 601914CB2352F2B5007ACE81 /* TupleN.swift */; };
+		606B29EA23607607002334B2 /* Divide.swift in Sources */ = {isa = PBXBuildFile; fileRef = 606B29E923607607002334B2 /* Divide.swift */; };
+		606B29EB2360763C002334B2 /* SemigroupalLaws.swift in Sources */ = {isa = PBXBuildFile; fileRef = 600D30442352D03400F7B64B /* SemigroupalLaws.swift */; };
+		606B29EE2360765D002334B2 /* DivideLaws.swift in Sources */ = {isa = PBXBuildFile; fileRef = 606B29EC2360764D002334B2 /* DivideLaws.swift */; };
+		606B29F023607745002334B2 /* MonoidalLaws.swift in Sources */ = {isa = PBXBuildFile; fileRef = 606B29EF23607745002334B2 /* MonoidalLaws.swift */; };
+		606B29F22360776A002334B2 /* Monoidal.swift in Sources */ = {isa = PBXBuildFile; fileRef = 606B29F12360776A002334B2 /* Monoidal.swift */; };
```

Comment:
rebased and fixed all pbxproj things @truizlop

Reply:
Sorry, I think merging the other PR caused conflicts on this again. Could you update your branch?